code/go/0chain.net/sharder/chain.go

```diff
@@ -35,10 +34,7 @@ func SetupSharderChain(c *chain.Chain) {
 	sharderChain.RoundChannel = make(chan *round.Round, 1)
 	blockCacheSize := 100
 	sharderChain.BlockCache = cache.NewLRUCache(blockCacheSize)
-	transactionCacheSize := int(c.BlockSize()) * blockCacheSize
-	if transactionCacheSize > 5000 {
-		transactionCacheSize = 5000
-	}
+	transactionCacheSize := 5 * blockCacheSize
```

Comment:
irrespective of the blocksize, we are setting the transaction cache size as 5 times the block cache size (which is 500). Why 5 times?

Reply:
I discussed this with @dabasov. He said, that in general transactionCacheSize is not mandatory to be calculated in the way it did before. So there may be any constant